pkg/lib/cockpit-components-password.scss

```diff
@@ -5,3 +5,7 @@
     display: none;
   }
 }
+
+#password-error-message .pf-c-helper-text__item-text {
+    color: var(--pf-c-form__helper-text--m-error--Color) !important;
```

Comment:
Does this really need an `!important`? It is already using an ID (which you should also try to avoid, but less so than `!important`).

Also, why not just use the error variant?

For example, here's the error variant with an icon:

```jsx
    <HelperText>
      <HelperTextItem variant="error" hasIcon>
        This is error helper text
      </HelperTextItem>
    </HelperText>
```

https://www.patternfly.org/v4/components/helper-text